edx_proctoring/tests/test_api.py

```diff
@@ -499,6 +500,233 @@ def test_remove_allowance_for_user(self):
         remove_allowance_for_user(student_allowance.proctored_exam.id, self.user_id, self.key)
         self.assertEqual(len(ProctoredExamStudentAllowance.objects.filter()), 0)
 
+    @ddt.data(
+        (
+            ADDITIONAL_TIME,
+            '30',
+            '30',
```

Comment:
Please add test cases with non-int to see the behavior of the function when a non-int is passed into it?
I know in production, passing in int for now is improbable. But we should also ensure future development of this repo do not use the function with non-int `ADDITIONAL_TIME`.